docs/adr/0001-mobile-app-architecture.md

```diff
@@ -0,0 +1,107 @@
+# Build Relay Mobile App with [Mobile Web | Qt | React Native | WebView ]
+
+- Status: proposed
+- Deciders: Luke Crouch, Emmett Lynch, Maxx Crawford
+- Date: pending
+
+Technical Spike:
+
+- [MPP-2633](https://mozilla-hub.atlassian.net/browse/MPP-2633)
+
+## Context and Problem Statement
+
+Make it just as easy to use your Firefox Relay phone number as your real phone number.
+See Google Voice as an example.
+[Firefox Relay Mobile Problems doc](https://docs.google.com/document/d/14wYnem-oPPA4_tjp018U3SyHpcdyKodDBbreRBkDd24/edit)
+
+## Decision Drivers
+
+- We are concerned that the Relay Phone experience is causing churn.
+
+## Considered Options
+
+- 2 mobile apps in Swift (iOS) and Kotlin (Android)
+- Re-use C++/Qt code-base from VPN
+- Mobile Web
+- React Native
+- React Native WebView
+
+## Decision Outcome
+
+Chosen option: "[option 1]", because [justification. e.g., only option, which meets k.o. criterion decision driver | which resolves force force | … | comes out best (see below)].
+
+### Positive Consequences <!-- optional -->
+
+- [e.g., improvement of quality attribute satisfaction, follow-up decisions required, …]
+- …
+
+### Negative Consequences <!-- optional -->
+
+- [e.g., compromising quality attribute, follow-up decisions required, …]
+- …
+
+## Pros and Cons of the Options <!-- optional -->
+
+### 2 mobile apps in Swift (iOS) and Kotlin (Android)
+
+Build and maintain 2 distinct code-bases and repos.
+Example: [Signal](https://github.com/signalapp/)
+
+- ❇️ Good, because the apps are in the app stores.
+- ❇️ Good, because each platform can have the "purest" experience possible.
```

Comment:
"purest" is a bit too unspecific too my liking. How about something like:

```suggestion
- ❇️ Good, because each platform optimises for these technologies, so our apps would by definition have no artificial restrictions that do not apply to other apps
```

And possibly performance/responsiveness plays a role there too to create such a "pure" experience?